index.html

```diff
@@ -1,182 +1,185 @@
+                <li id="title">Languages</li>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <img src="images/ic_languages_mob.svg" alt="testimage" class="hashtag">
+            </ul>
+            <ul class="fram">
+                <li id="title">Frameworks</li>
+                <li>React.js</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <img src="images/Rectangle 18.png" alt="rectangle18" class="rectangle18">
+                <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
+            </ul>
+            <ul class="skills">
+                <li id="title">Skills</li>
+                <li>Database Management</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Version Control</li>
+                <li>Web Development</li>
+                <img src="images/light.png" alt="light-bulb" class="bulb">
+            </ul>
 
-          <div class="get-contact">
-            <img src="images/Shape (1).png" alt="image4" class="image4">
-            <img src="images/Shape (2).png" alt="image5" class="image5">
-            <h1>Get Started</h1>
-            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
-            <form class="contact-form">
-            <div>
-              <label id="name-label">Your name</label>
-              <br>
-              <input id="name"  type="text" placeholder="Ricardo"/>
-            </div>
-            <div>
-              <label id="email-label">Your email</label>
-              <br>
-              <input id="email" type="email" placeholder="rj_egas@..."/>
-            </div>
-            <div>
-              <label>Your message</label>
-              <br>
-              <textarea name="comments" id="comment-box" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+            <div class="get-contact">
+                <img src="images/Shape (1).png" alt="image4" class="image4">
+                <img src="images/Shape (2).png" alt="image5" class="image5">
+                <h1>Get Started</h1>
+                <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
+                <form class="contact-form">
+                    <div>
+                        <label id="name-label">Your name</label>
+                        <br>
+                        <input id="name" type="text" placeholder="Ricardo" />
+                    </div>
+                    <div>
+                        <label id="email-label">Your email</label>
+                        <br>
+                        <input id="email" type="email" placeholder="rj_egas@..." />
+                    </div>
+                    <div>
+                        <label>Your message</label>
```

Comment:
- [ ] Kindly ensure that the `color`, `font-size` and `font-weight` of the form labels are similar to that of the Figma design :+1: So they look your design looks similar to the Figma design :+1: 
- [ ] From the images below, you can see that your design doesn't have similar spacing between the input elements as seen in the Figma design. Kindly implement this as well so your design looks similar to the Figma design:+1: 

| Your Design | Figma Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/229937931-9e4cc0ef-cdf8-4db1-9dd8-6ff06830bfc6.png) | ![image](https://user-images.githubusercontent.com/102699958/229937909-b99e2f75-8479-4c55-80b1-d8040d8a3a9e.png) |